libraries/engage/cookie-consent/components/CookieConsentDetail/index.jsx

```diff
@@ -0,0 +1,116 @@
+import React, { useCallback, useState } from 'react';
```

Comment:
the page is not very accessible right now. it's not possible to navigate to the switches with a screen reader. but i'd propose to take care about that after the first refactoring is applied.

Reply:
we don't have any accessibility rules applied anywhere, but ok..